_migration-assistant/getting-started-data-migration.md

```diff
@@ -2,18 +2,20 @@
 layout: default
 title: Getting started with data migration
 nav_order: 10
+redirect_from:
+  - /upgrade-to/upgrade-to/
+  - /upgrade-to/snapshot-migrate/
 ---
```

Comment:
Right now, steps have a mix of verbs in imperative and gerund forms. All steps should have verbs in imperative form.

Reply:
IMO, steps should be in the imperative form, so "Configure and deploy" rather than "Configuring and deploying". cc: @natebower

Reply:
Switched steps to imperative form.